src/main/resources/data/nyakomod/recipes/monitor.json

```diff
@@ -0,0 +1,26 @@
+{
+  "type": "minecraft:crafting_shaped",
+  "pattern": [
+    "GGG",
+    "RIR",
+    "DDD"
+  ],
```

Comment:
this looks like a fair and balanced crafting recipe!

Reply:
D